### DISCUSSION TITLE
Connected was thrown. Idk how to fix it.

### DISCUSSION THREAD

**Discussion by HukumaSV** [vpnhood/VpnHood#112] [Q&A]
Hello, vpnhood community.

I ran into an unusual problem.

I have a VPN server running on a computer with standard settings and server binding to console.vpnhood.com
In the web console, I created a private Access Token and entered the vh key to the mobile VPN client.

Since my network uses wi-fi, I did port forwarding on the router interface on port 443. Disabled firewall on computer and router.

In the server terminal console, when trying to connect from the phone, I see logs that the client is trying to connect to the server, but after a couple of seconds I get a notification on the phone of this nature: coonection was thrown.

I tried to set up the server in many ways, including setting up with Github and just a FAQ.
Still, I think the problem is in my router or I misconfigured NAT, but I'm not completely sure.

If someone faced such a problem, please write a solution, I will be very grateful.
Logs will be posted later

Thanks

**Comment by bobvhood** [comment]
Hi, Welcome here.
Is your server behind NAT, or your client (Mobile WIFI)?

**Reply by HukumaSV** [reply]
Hello.

The home network consists of a router and a computer, that is, an open network goes to the router - here I set up NAT and then Wi-fi is distributed to the computer.
From the phone I use the Internet of my mobile provider
NAT configured in such a way that external requests are redirected to the local address of the computer on port 443.
For example, the server starts with the local address 192.168.1.33 on port 443 and, as it were, should go further into the open network.  My mobile client connect to "white"-address, is that 178.204.*n*.*n*

**Reply by bobvhood** [reply]
Can you share the log file of your client?

**Reply by HukumaSV** [reply]
[LogVPN.txt](https://github.com/vpnhood/VpnHood/files/8335256/LogVPN.txt)

**Reply by HukumaSV** [reply]
Did everything again according to the instructions.  
Launched the server with default settings - appeared on the web console.  It contains 2 addresses: one of them is local, listened on port 443, the other is my white address, but it is not listening by default, I did not touch the value.  On the Wi-fi router, I configured NAT to let port 443 enter the network for all addresses. 
Next, in the Access Tokens menu, I created an access token, entered the vh key in the VPNhood mobile client.

**Reply by bobvhood** [reply]
> [LogVPN.txt](https://github.com/vpnhood/VpnHood/files/8335256/LogVPN.txt)

Your server 443 port is unreachable. As you mentioned it should be for NAT config. Unfortunately I really can't help you with what's wrong on your NAT config.
